gxa/src/test/java/uk/ac/ebi/atlas/experimentpage/json/opentargets/EvidenceServiceIT.java

```diff
@@ -43,23 +53,38 @@
     private EvidenceService<MicroarrayExpression, MicroarrayExperiment, MicroarrayRequestContext, MicroarrayProfile>
             subject;
 
-    @Before
-    public void setUp() {
+    @BeforeAll
+    void populateDatabaseTables() {
+        ResourceDatabasePopulator populator = new ResourceDatabasePopulator();
+        populator.addScripts(new ClassPathResource("fixtures/experiment-fixture.sql"));
+        populator.execute(dataSource);
+    }
+
+    @AfterAll
+    void cleanDatabaseTables() {
+        ResourceDatabasePopulator populator = new ResourceDatabasePopulator();
+        populator.addScripts(new ClassPathResource("fixtures/experiment-delete.sql"));
+        populator.execute(dataSource);
+    }
+
+    @BeforeEach
+    void setUp() {
         subject = new EvidenceService<>(microarrayProfileStreamFactory, dataFileHub, "test");
     }
 
     @Test
-    public void organismPartIsIncludedIfAvailable() {
+    void organismPartIsIncludedIfAvailable() {
         ImmutableList.Builder<JsonObject> listBuilder = ImmutableList.builder();
-        MicroarrayExperiment experiment = (MicroarrayExperiment) experimentTrader.getPublicExperiment("E-GEOD-12685");
-        subject.evidenceForExperiment(experiment, contrast -> {
-            MicroarrayRequestPreferences requestPreferences = new MicroarrayRequestPreferences();
-            requestPreferences.setFoldChangeCutoff(0.0);
-            requestPreferences.setCutoff(1.0);
-            requestPreferences.setHeatmapMatrixSize(1000);
-            requestPreferences.setSelectedColumnIds(ImmutableSet.of(contrast.getId()));
-            return new MicroarrayRequestContext(requestPreferences, experiment);
-        }, listBuilder::add);
+        MicroarrayExperiment experiment = (MicroarrayExperiment) experimentTrader.getPublicExperiment("E-GEOD-43049");
```

Comment:
It looks like this test escaped parameterization and it still has hard-coded experiment accessions. Unless it's changed in a later commit?